### ISSUE TITLE
Window position change after opening tab.

### ISSUE THREAD

**Issue by collmomo** [Turbo87/sidebar-v2#17]
On firefox 35.0.1, using navbar-top-fixed from bootstrap as a top menu, having a map underneath in a row div. When clicking the tabs, my window position is moved down. Leaflet.
```html

<header class="header clearfix navbar navbar-fixed-top" style="padding:0; background-color: #484848; margin-bottom: 0px; border-radius: 0px;z-index: 3000;" >
  <div class="container" style='margin-left: 0px;'>
    <div class="row">
    </div> 
  </div>
</header>
    <div class="row">
       <div class="col-md-12" style="padding:0px; margin-top: 65px;">
            <div id="map" style="height:600px" ></div>
    <!-- **************************************** MAP TABS ***************************************** -->
     <div id="sidebar" class="sidebar collapsed" style="top:85px;">
        <!-- Nav tabs -->
        <ul class="sidebar-tabs" role="tablist">
            <li><a href="#home" role="tab"><i class="fa fa-bars"></i></a></li>
            <li><a href="#profile" role="tab"><i class="fa fa-user"></i></a></li>
            <li><a href="#messages" role="tab"><i class="fa fa-envelope"></i></a></li>
            <li><a href="#settings" role="tab"><i class="fa fa-gear"></i></a></li>
        </ul>

        <!-- Tab panes -->
        <div class="sidebar-content active">
            <div class="sidebar-pane" id="home">
                <h1>sidebar-v2</h1>

                <p>A responsive sidebar for mapping libraries like <a href="http://leafletjs.com/">Leaflet</a> or <a href="http://openlayers.org/">OpenLayers</a>.</p>

                <p class="lorem">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.</p>

                <p class="lorem">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.</p>

                <p class="lorem">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.</p>

                <p class="lorem">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.</p>
            </div>
            <div class="sidebar-pane" id="profile"><h1>Profile</h1></div>
            <div class="sidebar-pane" id="messages"><h1>Messages</h1></div>
            <div class="sidebar-pane" id="settings"><h1>Settings</h1></div>
        </div>
    </div>
            </div>

    </div>
'''

**Comment by collmomo** [comment]
add    e.preventDefault(); at line 125.

**Comment by Turbo87** [comment]
@collmomo do you have a [jsfiddle](http://jsfiddle.net/) or something like that of your problem so that I can try to reproduce it? if that last comment is the fix then it would be great if you could prepare a pull request with that change.

**Comment by collmomo** [comment]
http://jsfiddle.net/b8b3Le65/
That jsfiddle is based on your leaflet example. If you scroll down the html view like this 
![screen shot 2015-01-31 at 14 11 06](https://cloud.githubusercontent.com/assets/1113095/5989350/0761ec5e-a953-11e4-9f4a-006a567b8d57.png)
and then try on clicking the settings icon, the window will become like this
![screen shot 2015-01-31 at 14 12 05](https://cloud.githubusercontent.com/assets/1113095/5989361/2e576c80-a953-11e4-887d-98f2778476e5.png)
Basically, the windows get repositioned, if you uncomment e.preventDefault(); at line 125, it will no longer reposition the windows.

**Comment by Turbo87** [comment]
I think this is related to #9. This library was intended to be used in full-screen applications, and not on pages that use scrolling since the sidebar widths wouldn't really fit well in those cases, but I guess solving #9 would also help your problems.
